nixos/modules/services/web-apps/miniflux.nix

```diff
@@ -96,12 +96,18 @@ in
         ++ lib.optionals cfg.createDatabaseLocally [ "postgresql.service" "miniflux-dbsetup.service" ];
 
       serviceConfig = {
-        ExecStart = "${cfg.package}/bin/miniflux";
+        Type = "notify";
+        ExecStart = lib.getExe cfg.package;
         User = "miniflux";
         DynamicUser = true;
         RuntimeDirectory = "miniflux";
         RuntimeDirectoryMode = "0750";
         EnvironmentFile = cfg.adminCredentialsFile;
+        WatchdogSec = 60;
+        WatchdogSignal = "SIGKILL";
```

Comment:
everything else looks fine here, but i do have to ask: why not stick with systemd's default of `SIGABRT` here? does miniflux not handle this well?` SIGKILL` shouldn't be really be used in most situations

Reply:
I copied this from upstreams systemd unit: https://github.com/miniflux/v2/blob/a631bd527d73b5b8d8e85343a1d5fe438e89dad7/packaging/systemd/miniflux.service#L26
I don't know how it behaves when using SIGABRT, I'll have a look.

Reply:
The go runtime prints the stack trace of all goroutines and dumps registers.
Looks like it works fine when using SIGABRT.

Reply:
> I copied this from upstreams systemd unit:

that's good enough for me. thanks!